easybuild/easyconfigs/p/psmpi/psmpi-5.1.5-1-GCC-4.9.3.eb

```diff
@@ -0,0 +1,27 @@
+name = 'psmpi'
+version = '5.1.5-1'
+
+homepage = 'https://github.com/ParaStation/psmpi2'
+description = """ParaStation MPI is an open source high-performance MPI 3.0 implementation,
+based on MPICH v3. It provides extra low level communication libraries and integration with 
+various batch systems for tighter process control.
+"""
+
+toolchain = {'version': '4.9.3', 'name': 'GCC'}
+
+toolchainopts = {'defaultopt': True}
+
+sources = ['%(version)s.tar.gz']
+source_urls = ['https://github.com/ParaStation/psmpi2/archive/']
+
+dependencies = [
+    ('pscom', '5.0.48-1', '', True),
+]
+
+hiddendependencies = [
+    ('pscom', '5.0.48-1', '', True),
```

Comment:
That's arguable. Sure, one site might want to expose every single module, but that's probably not something you want to do, right? In this particular case `pscom` is a library that is used exclusively by `psmpi` AFAIK, so I can't think of a reason to don't hide it.